catalogue/webapp/pages/item.js

```diff
@@ -50,41 +176,53 @@ const IIIFCanvasThumbnail = ({
   );
 };
 
-const Pagination = ({
+const MainXofY = ({
+  currentPage,
+  totalPages,
+}: PaginatorRenderFunctionProps) => (
+  <span
+    className={classNames({
+      'main-x-of-y': true,
+      [spacing({ s: 1 }, { margin: ['left', 'right'] })]: true,
+      [font({ s: 'LR3' })]: true,
+    })}
+  >
+    {currentPage} of {totalPages}
+  </span>
+);
+
+const PaginatorButtons = ({
   currentPage,
   totalPages,
   prevLink,
   nextLink,
 }: PaginatorRenderFunctionProps) => {
   return (
-    <div
-      className={classNames({
-        'flex flex--v-center flex--h-center': true,
-        [spacing({ s: 1 }, { margin: ['top', 'bottom'] })]: true,
-      })}
-    >
-      {prevLink && (
-        <NextLink {...prevLink} prefetch scroll={false}>
-          <a>
-            <Control type="light" icon="arrow" extraClasses="icon--180" />
-          </a>
-        </NextLink>
-      )}
-      <span
+    <div className="paginator-buttons">
+      <div
         className={classNames({
-          [spacing({ s: 1 }, { margin: ['left', 'right'] })]: true,
-          [font({ s: 'LR3' })]: true,
+          'flex flex--v-center flex--h-center': true,
         })}
       >
-        {currentPage} of {totalPages}
-      </span>
-      {nextLink && (
-        <NextLink {...nextLink} prefetch scroll={false}>
-          <a>
-            <Control type="light" icon="arrow" extraClasses="icon" />
-          </a>
-        </NextLink>
-      )}
+        {prevLink && (
+          <NextLink {...prevLink} prefetch scroll={false}>
```

Comment:
Prefetching seems like overkill as we can probably hijack these links and not make any requests to the server as we have all the data we need once we've loaded the page.